apps/academy/src/pages/tracks/index.tsx

```diff
@@ -11,10 +11,10 @@ const TracksPage: NextPage & { getLayout?: (page: ReactElement) => ReactElement
   const { data: allTracksData } = api.tracks.getAll.useQuery();
 
   return (
-    <div className="flex h-full w-full flex-col space-y-10 overflow-hidden bg-black lg:h-screen lg:max-h-screen lg:flex-row">
+    <div className=" flex h-screen w-full flex-col space-y-10 overflow-hidden bg-black lg:flex-row">
```

Comment:
@rohann06 Can you check the TracksPage background image? it's broken

![image](https://github.com/user-attachments/assets/2abf3817-94ae-42d2-8854-77e8f6ab0091)

should be something like this:
![image](https://github.com/user-attachments/assets/724c9b60-6cc1-421d-8608-092570ed7247)